pkg/apis/nats/v1alpha2/zz_generated.deepcopy.go

```diff
@@ -481,6 +482,16 @@ func (in *ServiceRoleSpec) DeepCopy() *ServiceRoleSpec {
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *TLSConfig) DeepCopyInto(out *TLSConfig) {
 	*out = *in
+	if in.CipherSuites != nil {
```

Comment:
What Go version do you guys have to support? `TLSConfig` has `.Clone()` for quite a bit now, so you may not need custom deep copy.

Reply:
This is generated code from k8s Ivan so we just apply as is... 🙈 but maybe the code generation tool will eventually support it.

Reply:
Yeah, you can split the PR into the two distinct commits; I did a generate first, committed that to have a baseline; then made my changes, committed, and then generated again, no diff -- my guess being that an inline simple type needed no extra logic.